man/systemd.exec.xml

```diff
@@ -1636,9 +1636,68 @@
         <literal>dumped</literal>. <varname>$EXIT_STATUS</varname> contains the numeric exit code formatted as string
         if <varname>$EXIT_CODE</varname> is <literal>exited</literal>, and the signal name in all other cases. Note
         that these environment variables are only set if the service manager succeeded to start and identify the main
-        process of the service.</para></listitem>
-      </varlistentry>
+        process of the service.</para>
+
+        <table>
+          <title>Summary of possible service result variable values</title>
+          <tgroup cols='3'>
+            <colspec colname='result' />
+            <colspec colname='status' />
+            <colspec colname='code' />
+            <thead>
+              <row>
+                <entry><varname>$SERVICE_RESULT</varname></entry>
+                <entry><varname>$EXIT_STATUS</varname></entry>
+                <entry><varname>$EXIT_CODE</varname></entry>
+              </row>
+            </thead>
+
+            <tbody>
+              <row>
+                <entry valign="top"><literal>exit-code</literal></entry>
+                <entry valign="top"><literal>exited</literal></entry>
+                <entry><literal>0</literal><sbr/><literal>1</literal><sbr/><literal>2</literal><sbr/><literal
+                >3</literal><sbr/>…<sbr/><literal>255</literal></entry>
```

Comment:
On purpose ;) Otherwise the line would be really long, and inside of a tag is the only place where docbook doesn't screw up formatting. I can make it one long line if you prefer.